exercises/practice/two-bucket/.docs/instructions.md

```diff
@@ -0,0 +1,46 @@
+# Instructions
+
+Given two buckets of different size and which bucket to fill first, determine how many actions are required to measure an exact number of liters by strategically transferring fluid between the buckets.
+
+There are some rules that your solution must follow:
+
+- You can only do one action at a time.
+- There are only 3 possible actions:
+  1. Pouring one bucket into the other bucket until either:
+     a) the first bucket is empty
+     b) the second bucket is full
+  2. Emptying a bucket and doing nothing to the other.
+  3. Filling a bucket and doing nothing to the other.
+- After an action, you may not arrive at a state where the starting bucket is empty and the other bucket is full.
+
+Your program will take as input:
+
+- the size of bucket one
+- the size of bucket two
+- the desired number of liters to reach
+- which bucket to fill first, either bucket one or bucket two
+
+Your program should determine:
+
+- the total number of actions it should take to reach the desired number of liters, including the first fill of the starting bucket
+- which bucket should end up with the desired number of liters - either bucket one or bucket two
+- how many liters are left in the other bucket
+
+Note: any time a change is made to either or both buckets counts as one (1) action.
```

Comment:
This is probably brought in from upstream, but this sentence seems malformed.  Perhaps the word "it" is missing from before "counts".

```suggestion
Note: any time a change is made to either or both buckets it counts as one (1) action.
```

Reply:
Yes upstream: https://github.com/exercism/problem-specifications/blob/0f7a7f8fef836d4271d548565666dcc3a54534d4/exercises/two-bucket/description.md?plain=1#L29